test/acceptance/aggregation.js

```
@@ -690,6 +694,66 @@ describe('aggregation', function () {
                 });
             });
 
+            it('use default aggregation by setting `aggregation: true`', function (done) {
```

What about adding a test to validate the contrary `aggregation: false`? 

Even I'd add a test to validate `aggregation: 'false'` should not work

---

We already had one for aggregation: false but was ineffective because the query used had fewer rows than the threshold, so it was not aggregated, but not because of the false. Now it's fixed with the `test_table_200k` table.

For the string case I think we should not only add tests, but change the behaviour: now strings (anything not an object or boolean) are just ignored, as if aggregation was not present, and I think it would be better to return a validation error. What do you think?

---

Fair enough! :)